### PR TITLE
ZK-3879: listbox autopaging does not work with external paginal

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zul/src/archive/web/js/zul/mesh/MeshWidget.js [hunk 1]
```diff
@@ -282,7 +282,8 @@ it will be useful, but WITHOUT ANY WARRANTY.
 				pgibHgh = pgib.offsetHeight;
 			var withPgibMax = max + pgibHgh;
 			if (top <= withPgibMax) { //row not exceeds the height, estimate
-				var totalsz = wgt.paging.getTotalSize(),
+				var paging = (wgt.paging) ? wgt.paging : wgt.getPaginal();
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zul/src/archive/web/js/zul/mesh/MeshWidget.js]: ok

[the remark this answers is not in the document]
Comment (klyvechen) [reply zul/src/archive/web/js/zul/mesh/MeshWidget.js]: ok

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**